packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -956,4 +970,40 @@ void main() {
     );
     EditableText.debugDeterministicCursor = false;
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
+
+  testWidgets('password briefly does not show last character on Android if turned off', (WidgetTester tester) async {
```

Comment:
This mentions Android but it's run on the current platform?  Shouldn't it fail on iOS?

Reply:
Renamed the test